fiona/env.py

```diff
@@ -538,7 +538,6 @@ def some_func(foo=None):
     reason = '\n{0}'.format(reason) if reason else reason
 
     def decorator(f):
-        @wraps(f)
```

Comment:
`wraps` wasn't breaking anything before. Can you explain what has changed @snowman2 ?

Reply:
The difference here is that it is being used on a class method instead of a function. It works fine for functions. However, when being applied to a class method, it breaks in Python 2 [see build logs](https://travis-ci.org/Toblerity/Fiona/builds/485700774) This issue was resolved in Python 3.2 [see here](https://bugs.python.org/issue3445). Definitely open to ideas for a better solution.